platform/forms/bundle.js

```diff
@@ -82,10 +88,21 @@ define([
                         "templateLinker",
                         "controls[]"
                     ]
+                },
+                {
+                    "key": "mctClickElsewhere",
```

Comment:
This directive is already defined in [commonUI/general](https://github.com/nasa/openmct/blob/ec4fe8efb39e4decb869edde9c6c61393c32ec6d/platform/commonUI/general/bundle.js#L331-L337); I don't think there is a reason you should need to redefine it here.